src/library/scala/jdk/DoubleAccumulator.scala

```diff
@@ -236,6 +237,7 @@ final class DoubleAccumulator
   }
 
   /** Copies the elements in this `DoubleAccumulator` into an `Array[Double]` */
+  @nowarn // cat=lint-overload see toArray[B: ClassTag]
   def toArray: Array[Double] = {
```

Comment:
This suggests a limitation of "manual specialization", namely the change in signature.

Would I ever want to `acc.toArray[Any]`?

Not sure yet whether to suppress for this case? or is this exactly what the lint is for?